### DISCUSSION TITLE
Add exported apps to new tenant.

### DISCUSSION THREAD

**Discussion by Bono-Boy** [Micke-K/IntuneManagement#137] [Q&A]
I have all the win32 apps in one folder. How do I import that into a new tenant? Now only the configurations are added.

**Comment by Micke-K** [comment]
Hello

Have a look at [Discussion 58](https://github.com/Micke-K/IntuneManagement/discussions/58). 

Quick guide:

- Export apps
- Configure folder in Settings
- Login to new tenant
- Import one app to make sure it works
- Import the rest

There is no API for decrypted intunewin files. That's why the script needs to have the path where the original files are located. 

I do recommend to import **without** assignments, unless you know 100% what will happen. It can have undesirable results eg recreate all groups from Tenant 1, push apps to devices without testing etc.

Cheers!

**Comment by Bono-Boy** [comment]
Thanks for the swift response. I found the problem why it wasn't importing the applications.
When exported the applications. The app package file had a different name the the one in the apps folder.
![image](https://github.com/Micke-K/IntuneManagement/assets/95429653/afccb5d1-3c7b-4026-80dd-d8adad9dc890)
When importing. This file wasn't there but it was 7z2201-x64.intunewin. Once i changed the name to 7z2201-x64.intunewin in the json. All was fine.
![image](https://github.com/Micke-K/IntuneManagement/assets/95429653/0799ea3d-0e9c-4582-a284-75ac9ef6c891)

**Comment by Micke-K** [comment]
Thank you for the update!

I like issues whit an easy explanation :) 

Cheers!
